php/libraries/SinglePointLogin.class.inc

```diff
@@ -356,7 +361,8 @@ class SinglePointLogin
             }
 
             if ($row['Active'] == 'N'
-                || $this->disabledDueToInactivity($username, 365)
+                || ($maxDaysInactive
```

Comment:
because of the not-so-predictable behaviour of intval(), I would instead do a more explicit check like `(!empty($maxDaysInactive) && is_numeric($maxDaysInactive))`

intval:
```
Return Values ¶
The integer value of value on success, or 0 on failure. Empty arrays return 0, non-empty arrays return 1.

The maximum value depends on the system. 32 bit systems have a maximum signed integer range of -2147483648 to 2147483647. So for example on such a system, intval('1000000000000') will return 2147483647. The maximum signed integer value for 64 bit systems is 9223372036854775807.

Strings will most likely return 0 although this depends on the leftmost characters of the string. The common rules of integer casting apply
```